.github/workflows/GPU.yml

```diff
@@ -0,0 +1,43 @@
+name: GPU
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
I guess ultimately we want it on stable too.

Reply:
yes, after the next release.

Reply:
You should add `stable` here then, it won’t trigger until the workflow is on that branch, but at least then we don’t forget.